packages/inspection-capture-web/src/PhotoCapture/hooks/usePhotoCaptureSightState.ts

```diff
@@ -162,10 +163,26 @@ export function usePhotoCaptureSightState({
   const [sightsTaken, setSightsTaken] = useState<Sight[]>([]);
   const { getInspection } = useMonkApi(apiConfig);
   const { handleError } = useMonitoring();
+  const analytics = useAnalytics();
+
+  const selectSight = (s: Sight) => {
+    const sightsNotTaken = captureSights.filter((sight) => !sightsTaken.includes(sight)) ?? [];
+    const nextSight = sightsNotTaken.at(sightsNotTaken.indexOf(s) + 1)?.label ?? 'last-sight';
```

Comment:
instead of `'last-sight'` maybe pass `null`?

Reply:
i will add an other tag for the label, easier to filter out by part in posthog